src/utils/request.util.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * 
+ * @param {object} page 
+ * @returns {object} offset and limit 
+ * @description it returns offset and limit when it is passed a page
+ */
+export const dbPage = (page) => {
```

Comment:
give this function a more descriptive name

Reply:
yes sure